tests/LPs/chat-feature.test.js

```diff
@@ -0,0 +1,49 @@
+
+const { test, expect } = require('@playwright/test');
+
+//Config
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://meet.swordhealth.com/');
+});
+
+//Main test: Chat feature  
+test.describe('Chat feature', () => {
+
+  //Test 1: Confirm element inside chat feature
+  test('When user clicks button, element should display', async ({ page }) => {
+    
+    // User clicks on the Chat button
+    await page.getByTestId('launcher').click();
+
+    // 2 seconds wait to open modal
+    await page.waitForTimeout(2000);
+
+    // User confirms element is visible on chat form
+    await expect(page.getByTestId('button-ok')).toBeVisible();
```

Comment:
The chat itself is in a different frame, so we need to locate that one as well.
We don't need the timeout since `await` itself is asynchronous.
Playwright has a default timeout of 3s - it will wait for 3s before returning an error.

```suggestion
    // Chat is in a different frame than the button
    // Locate the chat frame
    const chatFrame =  await page.frameLocator('#webWidget');

    // User confirms element is visible on chat form
    await expect(chatFrame.getByTestId('button-ok')).toBeVisible();
```